File: app/src/main/java/amagi82/modularcharactersheetcreator/utils/ScreenSize.java
package amagi82.modularcharactersheetcreator.utils;

import android.content.Context;
import android.graphics.Point;
import android.view.Display;
import android.view.WindowManager;

public class ScreenSize {

    private int width;
    private int height;

    public ScreenSize(Context context) {
        WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
        Display display = wm.getDefaultDisplay();
        Point size = new Point();
        display.getSize(size);
        width = size.x;
        height = size.y;
    }

    public int getWidth(){
        return width;
    }

    public int getHeight(){
        return height;
    }
}

File: app/src/main/java/amagi82/modularcharactersheetcreator/ui/_extras/utils/ScreenSize.kt
package amagi82.modularcharactersheetcreator.ui._extras.utils

import android.content.Context
import android.graphics.Point
import android.view.WindowManager

class ScreenSize(context: Context) {
    val width: Int
    val height: Int

    init {
        val display = (context.getSystemService(Context.WINDOW_SERVICE) as WindowManager).defaultDisplay
        val size = Point()
        display.getSize(size)
        width = size.x
        height = size.y
    }
}